File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.py
def solution(s):
    answer =[]
    cnt = 0
    tb = 0

    while True:
        if s  == '1':
            break
        tb += s.count("0")
        s = s.replace("0","")
        s = bin(len(s))[2:]
        cnt += 1
    answer = [cnt,tb]
    return answer

File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.js
function solution(s) {
    var cnt = 0;
    var zero = 0;
    var temp = s;
    while(temp.length > 1) {
        cnt++;
        zero += (temp.split("0")).length - 1;
        temp ="1" * (temp.length - ((temp.split("0")).length - 1))
        temp = temp.toString(2);
    }
    return [cnt,zero];
}